html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get" >
+      <label for="email">Email</label>
+      <div>
+        <input type="text" name="email" id="email">
+      </div>
+      <label for="coding options">Why is coding cool?</label>
```

Comment:
There should be no spaces in the values of `for` or `id` attributes, you should use `kebab-case` to connect words, each word should have a dash between it and the next word rather than a space.